src/main.sh

```diff
@@ -41,7 +34,9 @@ function _show_version {
 
 function _init_script {
   if [[ $# == 0 ]]; then
-    _incorrect_usage 'no input parameters provided.' 126
```

Comment:
Why did you change this?

Reply:
shellcheck correctly warns that the usage() function expects parameters but is never called with any. I have just submitted a different pull request based on the current master which passes the rest of the parameter list to the usage function rather than inline it. So you can forget this part of this submission.